src/e3/pytest.py

```diff
@@ -194,7 +194,16 @@ def fix_coverage_paths(origin_dir: str, new_dir: str, cov_db: str) -> None:
         old_coverage_data = CoverageData(old_cov_file.name)
         old_coverage_data.read()
         new_coverage_data = CoverageData(cov_db)
-        new_coverage_data.update(old_coverage_data, aliases=paths)
+        # Incompatibility with version 7.5.2 and 7.5.3 of the update() method.
```

Comment:
```suggestion
        # aliases argument has been replaced by map_path in coverage 7.5.3 release.
```

Reply:
Can we do the opposite? Make the call work by default with the new version of coverage and have a fallback for legacy versions?

Reply:
Done